### PR TITLE
add new mne python module

### LINKED ISSUE #187: Adding an MNE-EEG module
### Title

Adding an MNE-EEG module

### Topic keywords

mne

### Module Description

C﻿ourse Description

T﻿opics covered will include:

loading, filtering, and inspecting raw data
working with BIDS data
epoching and artifact correction
creating and visualizing evoked responses (ERP / ERF)
contrasting evoked responses of different experimental conditions
decoding neural responses (machine learning)
performing time-frequency analysis
estimating and visualizing cortical sources (source localization)
conducting a group analysis
Permutation test and cluster correction

### Key tools / technology included in this module

- Open-source
- Python-based
- Quality software (well maintained, not serving a single research project, clear feedback / bug report system)
- MNE-python

### Prerequisites

Introduction to Python / previous experience with Python

### Study outcomes

(https://github.com/Davi1990/mne_eeg_workshop)

### Estimated study time

_No response_

### Exercise examples

n/a

### References you would like to include

_No response_

### Things to check by the reviewers.

- [X] Use free, open source tools.
- [X] Python-based tool.
- [ ] Be of high quality.
- [ ] Have a broad appeal.

### PR DESCRIPTION
**Pull request by Davi1990**
Closes #187 (edit bu HTW to link PR to issue)

This is the initial pull request for adding the MNE-Python module to Brainhack-school

I wanted to thank you for taking the time to review my pull request. 

I believe this could be a great opportunity for us to exchange ideas and improve the materials together

I appreciate any feedback or suggestions you may have

Thank you again, and I look forward to working with you.

Cheers 

Dave

### CODE REVIEW

File: content/en/modules/mne_python/index.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+```
+pip install -r requirements.txt
+```
+
+ * Download the required data folder at: https://drive.google.com/drive/folders/1DO-dXfIXzGDzmgcWRMtYvX30ZECYzRYd?usp=sharing
+
+ * Download the raw file at: https://drive.google.com/file/d/1-RSyaXp2Chx0zLuaAnlgK8o1VMo3enx8/view?usp=share_link
+
+ * Start a new jupyter notebook
+ * Watch the video and  run the cells in the notebook
```
Comment (htwangtw) [inline content/en/modules/mne_python/index.md:61]:
```suggestion
 * Watch the video and run the cells in the notebook
```

File: content/en/modules/mne_python/index.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+       What is the frequency of the line noise?
+       Is there any bad channel?
+
+**Exercise 2** Import a new epoched file and generating Figure of both evoked response and principal components
+
+        Read the file 'sub-s01_task-faceFO_eeg.fif' as an epoched object
+        Make a joint plot of the time-series?
+        Create a variable with the array of the evoked data
+        Decompose the data using SVD (use the function numpy.linalg.svd)
+        Make a matplotlib figure with 1 row and 5 columns and plot the first 5 components using the function mne.viz.plot_topomap
```
Comment (htwangtw) [inline content/en/modules/mne_python/index.md:82]: This block might not render correctly. Please build the page locally and double check.

File: content/en/modules/mne_python/index.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+
+ * Follow up with Natasha Clarke to validate you completed the exercise correctly.
```
Comment (htwangtw) [inline content/en/modules/mne_python/index.md:85]: You might want to change the name...

**Review by htwangtw — DISMISSED** [review]

**Review by clarkenj — APPROVED** [review]
LGTM!

**Pull request merged**